.github/CODEOWNERS

```diff
@@ -0,0 +1,31 @@
+# Lines starting with '#' are comments.
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in the repo.
+* @pierre-sassoulas @DanielNoord @cdce8p @jacobtylerwalls @hippo91
+
+# Order is important. The last matching pattern has the most precedence.
+
+# @DudeNr33
+pylint/pyreverse/*    @DudeNr33
+tests/pyreverse/*    @DudeNr33
+
+# @pierre-sassoulas
+pylint/messages/*    @pierre-sassoulas
+
+# typing
+pylint/typing.py     @DanielNoord @cdce8p
+
+# multiprocessing
+# pylint/lint/parallel.py @doublethefish
+# tests/test_check_parallel.py @doublethefish
+
+# @yushao2
+tests/functional/u/unused/unused_private_member.py @yushao2
```

Comment:
I don't think we should add codeowners for tests alone. That's quite useless IMO.

Reply:
The issue is that the "checker" itself is really mixed inside a "stdlib" checker. I think we need to make checker more independent and hide the checkers from the users but I did not have this hindsight when we added the unused-private-member check.